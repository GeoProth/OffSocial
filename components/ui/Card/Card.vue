<template>
  <div class="ui-card">
    <slot />
  </div>
</template>
<script>
export default {
  name: "Card"
};
</script>
<style lang="stylus" scoped>
.ui-card {
  border-radius: 0.5rem
  background-color: #fafbfc
  color: black
  text-align: center
  input {
    appearance: none;
    margin: 0 1.5rem;
    height: 3.5rem;
    padding: 0.5rem 1.5rem;
    border-radius: 0.5rem;
    outline: none;
    border: none;
    background: #e8ebed;
  }
}
</style>
