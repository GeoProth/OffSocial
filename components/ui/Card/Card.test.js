import { shallowMount } from "@vue/test-utils";
import Card from "./Card";
describe("Card component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(Card);
  });
  test("component can render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
