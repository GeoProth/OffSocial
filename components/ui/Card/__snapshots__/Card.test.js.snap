// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component component can render 1`] = `<div class="ui-card"></div>`;
